src/web/src/components/Posts/GItHubInfoMobile.tsx

```diff
@@ -0,0 +1,51 @@
+import { createStyles, makeStyles, Theme, ListSubheader } from '@material-ui/core';
+import Repos from './Repos';
+import Issues from './Issues';
+import PullRequests from './PullRequests';
+import Commits from './Commits';
+import Users from './Users';
+import { filterGitHubUrls } from './GitHubInfo';
+
+type Props = {
+  ghUrls: string[];
+};
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      padding: '0',
+      display: 'flex',
+      flexDirection: 'column',
+      marginTop: '0',
+      lineHeight: '1',
+      [theme.breakpoints.up('lg')]: {
+        width: '21rem',
+      },
+      color: theme.palette.text.secondary,
+    },
+    GitHubInfoContainer: {
+      margin: '0 0 0 1rem',
+    },
+  })
+);
+
+const GitHubInfoMobile = ({ ghUrls }: Props) => {
+  const classes = useStyles();
+  const { repos, issues, pullRequests, commits, users } = filterGitHubUrls(ghUrls);
```

Comment:
let's cache the result of this function call with useMemo so that it doesn't get called every time the `Accordion` is open. 
```suggestion
  const { repos, issues, pullRequests, commits, users } = useMemo(
    () => filterGitHubUrls(ghUrls),
    [ghUrls]
  );
```
Can you do the same for `GithubInfo`?